site/content/docs/5.2/components/list-group.md

```diff
@@ -269,6 +269,31 @@ Place Bootstrap's checkboxes and radios within list group items and customize as
 </ul>
 {{< /example >}}
 
+{{< example >}}
```

Comment:
IMO maybe you could just display 3 radio buttons. With 5 checkboxes, 5 radio buttons, 5 checkboxes again + source code, it becomes long to scroll in the documentation.